### PR TITLE
Adding groupmsg slash command to documentation

### PR DESCRIPTION
**Pull request by jespino**
Wait until groupmsg command is merged mattermost/mattermost-server#7419

### CODE REVIEW

File: source/developer/slash-commands.rst [hunk 1]
```diff
@@ -36,6 +36,7 @@ Each Mattermost installation comes with some built-in slash commands that are re
     "/logout", "Log out of Mattermost", "/logout"
     "/me *{message}*", "Do an action", "/me Hello World"
     "/msg *{@username}* *{message}*", "Send a Direct Message to a user", "/msg @alice hello"
+    "/groupmsg *{@username,...}* *{message}*", "Sends a Group Message to the specified users", "/groupmsg @alice,@bob hello"
```
Comment (lindy65) [inline source/developer/slash-commands.rst]: Not sure whether there should be a space between `@alice,@bob...`?

Comment (jespino) [reply source/developer/slash-commands.rst]: Good catch. I have changed it. Probably is more clear that you can use spaces in this way, and with the "," is already clear that you can use it without spaces.

**Comment by jasonblais** [comment]
@jespino We should also update the table at https://github.com/mattermost/docs/blob/master/source/help/messaging/executing-commands.rst :)

**Comment by jespino** [comment]
Ups, I didn't see it. I'll add another PR.

**Pull request merged**